daemon/integration/integration_test.go

```diff
@@ -39,7 +41,7 @@ func TestRunSimpleCommand(t *testing.T) {
 	runId := assertRun("ignore", "running", "nil", statusReq[:], t)
 
 	// give the command time to finish running
-	time.Sleep(30 * time.Millisecond)
+	time.Sleep(100 * time.Millisecond)
```

Comment:
These changes seem completely unrelated to the other changes? Was this failing for you/travis-ci?